src/frontend/components/Standings/components/DriverInfoRow/DriverInfoRow.tsx

```diff
@@ -131,11 +135,9 @@ export const DriverInfoRow = ({
           {lastTimeString}
         </td>
       )}
-     {tireCompound !== undefined && (
-        <td>
-          <div className="flex items-center pr-1"> 
-           <Compound tireCompound={tireCompound} size="sm" />
-          </div>
+     {tireCompound !== undefined && carId && (
+        <td className="px-2">
+           <Compound tireCompound={tireCompound} carId={carId} size="sm" className="mr-2 flex-shrink-0" />
```

Comment:
Having the container lets it be centred better vertically so let's keep that. It might've been overwritten when rebasing.
```suggestion
       <div className="flex items-center pr-1"> 
           <Compound tireCompound={tireCompound} carId={carId} size="sm" />
        </div>   
```